drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c

```diff
@@ -187,6 +188,8 @@ static int bcmf_transmit(FAR struct bcmf_dev_s *priv,
       return -EIO;
     }
 
+  priv->bc_dev.d_buf = NULL;
```

Comment:
we can reuse cur_tx_frame to check whether devif_poll really transmit some packet.

Reply:
Done